tests/fixtures/rendering.yml

```diff
@@ -321,27 +321,6 @@
     equals:
       'h3.ecl-list-item__title': "Item title"
       'div.ecl-list-item__detail': "Item text"
-- array:
```

Comment:
Why was this test removed?

Reply:
As the pattern is using a `DateValueObject`, we need to deal with an object, not only an array.
This is why this test was removed from there and moved into `DateBlockPatternRenderingTest.php`